response/core/serializers.py

```diff
@@ -8,36 +8,60 @@
 from django.contrib.auth.models import User
 
 
-class ExternalUserSerializer(serializers.HyperlinkedModelSerializer):
-    owner = serializers.PrimaryKeyRelatedField(queryset=User.objects.all(), required=False)
-
+class ExternalUserSerializer(serializers.ModelSerializer):
     class Meta:
         model = ExternalUser
-        fields = ('app_id', 'external_id', 'owner', 'display_name')
+        fields = ("app_id", "external_id", "display_name")
```

Comment:
Previously we were returning the owner user primary key. I took it out, as it's basically meaningless, and we can more easily identify an External User by the other three (we have a UNIQUE constraint on them)